internal/testing/e2e/query_tickets_test.go

```diff
@@ -150,16 +150,14 @@ func TestTicketNotFound(t *testing.T) {
 }
 
 func TestBackfillFound(t *testing.T) {
-	tc := testcases.TestCase{
-		Name:         "OwnTest",
-		Pool:         &pb.Pool{},
-		SearchFields: nil,
+	for _, tc := range testcases.IncludedTestCases() {
+		tc := tc
+		t.Run("QueryBackfill/"+tc.Name, func(t *testing.T) {
+			if !returnedBackfillByQuery(t, tc) {
+				require.Fail(t, "Expected to find backfill in pool but didn't.")
+			}
+		})
```

Comment:
Hey @Laremere i could use some help here. I'm using the currently existing test cases for querying backfills. Do you think are useful scenarios which doesn't provide false positives?? Or should we revisit this??

Reply:
I'm not sure what you're asking?  These tests seem good to me.

Reply:
I was asking if current test cases are useful for testing this. But if you think the test cases are good for querying backfills, i'll leave them untouched